gnucash/import-export/qif-imp/qif-objects.scm

```diff
@@ -261,12 +263,29 @@
 (define qif-split:set-memo! 
   (record-modifier <qif-split> 'memo))
 
-(define qif-split:amount 
+(define qif-split:amount-internal
   (record-accessor <qif-split> 'amount))
 
+(define qif-split:uamount-internal
+  (record-accessor <qif-split> 'uamount))
+
+(define (qif-split:amount split)
+  (let ((uamount (qif-split:uamount-internal split))
+        (amount (qif-split:amount-internal split)))
+    (cond
+     ((number? amount) amount)
```

Comment:
This should be the *second* condition: It's always true if there's a `T` field even when the `T` field is wrong.